rest-api/src/main/java/gov/cms/qpp/conversion/api/services/DbService.java

```diff
@@ -19,4 +19,6 @@
 	CompletableFuture<Metadata> write(Metadata meta);
 
 	List<Metadata> getUnprocessedCpcPlusMetaData();
+
+	String getFileSubmissionLocationId(String uuid);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Document this public method by adding an explicit description. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUndocumentedApi)

Reply:
Oops... javadocs